submissions/NastjonkaK/OOP_exercise/index.js

```diff
@@ -0,0 +1,84 @@
+class Creature {
+  constructor(name, saying) {
+    this.name = name,
+    this.saying = saying
+  }
+}
+
+class Animals extends Creature {
+  constructor(name, gender, saying) {
+    super(name, saying);
+    this.gender = gender,
+    this.paws = 4
+  }
+
+  greeting() {
+    return `\ Hello! I am a ${this.type}, ${this.gender}. My name is ${this.name}. I have ${this.paws} paws and a tail. And I like to say: "${this.saying}".`;
+  }
+}
+
+class Human extends Creature {
+  constructor(name, saying) {
```

Comment:
Try to implement [default parameters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters) (for all instances).  Not all people have 2 hands and legs :crying_cat_face: 
example 
```
constructor(name, saying, hands=2, legs=2) {
  super(name, saying)
    this.type = "human",
    this.legs = hands,
    this.hands = legs
}
```